voronoi.cpp

```diff
@@ -139,6 +139,44 @@ void userfree(void* ctx, void* ptr) {
 	return memfree(ptr);
 }
 
+void Voronoi::relax_points(int iterations = 1) {
+	std::vector<jcv_point> new_points;
+	new_points.reserve(_points.size());
+	for (int j = 0; j < iterations; j++) {
+		Ref<VoronoiDiagram> diagram { memnew(VoronoiDiagram) };
```

Comment:
As far as I see, the creation of a godot class is not needed, is it? Below, you only make use of the jcv* functions and only operate on diagram->_diagram, not using any methods of the godot classes. Thus, I think using a jcv_diagram directly here (as stack variable as it is anyways only used in this scope) is the easiest solution with the least overhead.

Reply:
Thanks so much for the feedback - I'm not a c/c++ developer so I don't have a good intuition of these things. I've attempted to resolve your issue, please see my latest commit :)